ros/src/tl_detector/tl_detector.py

```diff
@@ -133,11 +191,20 @@ def get_light_state(self, light):
             int: ID of traffic light color (specified in styx_msgs/TrafficLight)
 
         """
-        # use rgb8 instead of bgr8 for detect
-        cv_image = self.bridge.imgmsg_to_cv2(self.camera_image, "rgb8")
 
-        #Get classification
-        light_state = self.light_classifier.get_classification(cv_image)
+
+        if self.config['is_site']:
```

Comment:
So, one thing I've been thinking about and don't want to get burned on is to have the 'is_site' models be the default, so if for some reason Udacity doesn't set it correctly we will be using the site models

Reply:
What do you mean? 
While I agree on your point it seems to me that with this code the "is_site" branch is not the default one. Anyway if everything work as expected we can refactor the code in a second stage, given the short timeline